links/link.go

```diff
@@ -150,8 +150,8 @@ func (ld *LinkDefinition) UnmarshalYAML(unmarshal func(interface{}) error) error
 	case LinkTypeBrief:
 		// brief link's endpoint format
 		var l struct {
-			Type      string `yaml:"type"`
-			LinkBrief `yaml:",inline"`
+			Type         string `yaml:"type"`
+			LinkBriefRaw `yaml:",inline"`
```

Comment:
I was not sure that LinkBrief should be renamed to LinkBriefRaw, since it is by default Raw and there is no `Resolve` to make it non-raw

Reply:
I think this way it will follow the schema, all text based struts are raw. Others are the regular structs.
Also mgmt-net does have a raw but no resolved  representation itself.
But linkbriefraw has to implement the interface rawlink otherwise we can not get it into the linkdefinition.
I think it is fine this way.

Reply:
okay let's ship it
the rest is ok